jute.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="470" height="60" class="svg"><title>badge-for-git. Style: 17</title><desc>char-height: 60 char_width : 30 font-size  : 50</desc><path fill="#CB0000" d="M30,0 30,0 a30,30 0 0,1 0,60 L30,60 a30,30 0 0,1 0,-60"></path><text fill="#FFF" x="30" y="49" font-family="DejaVu Sans Mono, Source code variable, monospace" font-size="50"></text><path fill="#CB0000" d="M100,0 100,0 a30,30 0 0,1 0,60 L100,60 a30,30 0 0,1 0,-60"></path><text fill="#FFF" x="100" y="49" font-family="DejaVu Sans Mono, Source code variable, monospace" font-size="50"></text><path fill="#CB0000" d="M170,0 290,0 a30,30 0 0,1 0,60 L170,60 a30,30 0 0,1 0,-60"></path><text fill="#FFF" x="170" y="49" font-family="DejaVu Sans Mono, Source code variable, monospace" font-size="50">Jutu</text><path fill="#CB0000" d="M360,0 360,0 a30,30 0 0,1 0,60 L360,60 a30,30 0 0,1 0,-60"></path><text fill="#FFF" x="360" y="49" font-family="DejaVu Sans Mono, Source code variable, monospace" font-size="50"></text><path fill="#CB0000" d="M430,0 430,0 a30,30 0 0,1 0,60 L430,60 a30,30 0 0,1 0,-60"></path><text fill="#FFF" x="430" y="49" font-family="DejaVu Sans Mono, Source code variable, monospace" font-size="50"></text></svg>
```

Comment:
I think this logo says "Jutu", instead of "Jute" :D

Reply:
Yes you are right I modified it on the fly :smile: I did not touch the JSON format, since it was already 8 spaces for indentation. It will be modified on the fly again :smile: